src/managers/LayoutManager.hpp

```diff
@@ -13,6 +13,7 @@ class CLayoutManager {
 
     bool         addLayout(const std::string& name, IHyprLayout* layout);
     bool         removeLayout(IHyprLayout* layout);
+    std::vector<std::pair<std::string, IHyprLayout*>> m_vLayouts;
```

Comment:
I'd rather have a function that returns data about loaded layouts. m_vLayouts should stay privat

Reply:
I'll just do whatever the plugin shit does. That uses unique_ptr and returns the raw pointers. I have some layout plugins that need to hold references to layouts (because doing name lookups constantly is dumb) and I'm pretty sure unique_ptr+raw is unsafe. Can we used shared_ptr here and for stuff that holds layout ptrs they can just make weak_ptrs? Or is there a better way to deal with that?

Reply:
return just the names